CandidateApp.ConsoleApp/Models/Candidate/Candidate.cs

```diff
@@ -0,0 +1,13 @@
+﻿// ------------------------------------------------
+// Copyright (c) MumsWhoCode. All rights reserved.
+// ------------------------------------------------
+
+namespace CandidateApp.ConsoleApp.Models.Candidate
```

Comment:
The folder should be in plural - Candidates

Reply:
Changed the folder name as well